packages/contracts/.cct/setup-fork.js

```diff
@@ -32,54 +16,107 @@ export const setupFork = async (
     // deploy template
     const {
         abi: templateAbi,
-        bytecode: templateBytecode,
-    } = require(`../out/RealityV3Oracle${chainId}.sol/RealityV3Oracle.json`);
-    const templateFactory = new ContractFactory(
-        templateAbi,
-        templateBytecode,
-        signer
-    );
-    const templateContract = await templateFactory.deploy();
-    await templateContract.deployed();
+        bytecode: { object: templateBytecode },
+    } = require(`../artifacts/RealityV3Oracle${chainId}.sol/RealityV3Oracle.json`);
+
+    const { contractAddress: templateAddress } =
+        await nodeClient.getTransactionReceipt({
+            hash: await walletClient.deployContract({
+                abi: templateAbi,
+                bytecode: templateBytecode,
+            }),
+        });
 
     // deploy arbitrator
     const {
         abi: arbitratorAbi,
-        bytecode: arbitratorBytecode,
-    } = require(`../out/TrustedRealityV3Arbitrator${chainId}.sol/TrustedRealityV3Arbitrator.json`);
-    const arbitratorFactory = new ContractFactory(
-        arbitratorAbi,
-        arbitratorBytecode,
-        signer
-    );
-    const arbitratorContract = await arbitratorFactory.deploy("{}", 0);
-    await arbitratorContract.deployed();
+        bytecode: { object: arbitratorBytecode },
+    } = require(`../artifacts/TrustedRealityV3Arbitrator${chainId}.sol/TrustedRealityV3Arbitrator.json`);
 
-    await new Contract(
-        arbitratorContract.address,
-        trustedRealityV3ArbitratorAbi,
-        signer
-    ).setDisputeFee(utils.parseEther("1"));
+    const { contractAddress: arbitratorAddress } =
+        await nodeClient.getTransactionReceipt({
+            hash: await walletClient.deployContract({
+                abi: arbitratorAbi,
+                bytecode: arbitratorBytecode,
+                args: ["{}", 0],
+            }),
+        });
 
-    // give us some wrapped native currency too
-    await new Contract(
-        WRAPPED_NATIVE_CURRENCY_ADDRESS[chainId],
-        wrappedNativeCurrencyAbi,
-        signer
-    ).deposit({
-        value: utils.parseEther("1"),
+    const arbitratorContract = getContract({
+        abi: arbitratorAbi,
+        address: arbitratorAddress,
+        publicClient: nodeClient,
+        walletClient: walletClient,
     });
 
+    // set a minimum arbitration fee
+    await arbitratorContract.write.setDisputeFee([parseUnits("1", 18)]);
+
+    // deploy test erc20 tokens
```

Comment:
Do we actually need these? They are not used in the Reality.eth template, I think it's a copy-paste thing from the ERC20 KPI token.

Reply:
I'd keep them since they can be used as collaterals when creating a campaign while developing in the playground.

Reply:
Ohhh ok. Even though this is really really hacky. This ONLY works because the ERC20 KPI token template manually adds the tokens on the default token list when running in dev mode. It relies on a dev-only logic present in another template.

Reply:
Oh, didn't think about it that way, if you think it's not "healthy" we can remove this.

Reply:
Nah I think it's ok, but maybe I'd add a FIXME or something. My fear is that we forget, because this is specific logic that only works when the oracle template is paired with the ERC20 KPI token template.